runtime/src/main/cpp/Memory.h

```diff
@@ -107,7 +107,7 @@ extern "C" {
 
 #define OBJ_RESULT __result__
 #define OBJ_GETTER0(name) ObjHeader* name(ObjHeader** OBJ_RESULT)
-#define OBJ_GETTER(name, ...) ObjHeader* name(__VA_ARGS__, ObjHeader** OBJ_RESULT)
+#define OBJ_GETTER(name, ...) RUNTIME_NOTHROW ObjHeader* name(__VA_ARGS__, ObjHeader** OBJ_RESULT)
```

Comment:
I'm not sure that's correct. For example, see `Kotlin_Array_get`
https://github.com/JetBrains/kotlin-native/blob/1aef9da517f5b56a31e213929e6416d0751e96ab/runtime/src/main/cpp/Arrays.cpp#L99-L103
It calls `boundCheck`
https://github.com/JetBrains/kotlin-native/blob/1aef9da517f5b56a31e213929e6416d0751e96ab/runtime/src/main/cpp/Arrays.cpp#L37-L44
which does throw an exception. And since Kotlin/Native exceptions are actually C++ exceptions:
https://github.com/JetBrains/kotlin-native/blob/1aef9da517f5b56a31e213929e6416d0751e96ab/runtime/src/main/cpp/Exceptions.cpp#L209-L218
this may result in `Kotlin_Array_get` terminating the program instead of throwing an exception.